source/__init__.py

```diff
@@ -0,0 +1 @@
+#do I need this file anymore?
```

Comment:
Yes otherwise some imports may not work (which also remind me that we need to stop import *)

Here is a very brief intro to modules: https://pythontips.com/2013/07/28/what-is-__init__-py/